.gitmodules

```diff
@@ -40,3 +40,6 @@
 [submodule "plugins/couchbase/libcouchbase-cxx"]
 	path = plugins/couchbase/libcouchbase-cxx
 	url = https://github.com/hpcc-systems/libcouchbase-cxx.git
+[submodule "libcouchbase"]
+	path = plugins/couchbase/libcouchbase
+	url = https://github.com/couchbase/libcouchbase.git
```

Comment:
We should fork the upstream repo and use that as a submodule - otherwise our buil can be broken by changes to the upstream repo (e.g. if they moved it...)